src/Misc/layoutbin/en-US/strings.json

```diff
@@ -447,6 +447,8 @@
   "SourceDirectoriesNotSpecified": "Source directories field is not specified. Specify valid source directories for code coverage reports to include highlighted source code and try again.",
   "SourceDirectoriesNotSpecifiedForMultiModule": "Source directories field is not specified. Source directories field is required for a multi module project for code coverage reports to include highlighted source code.",
   "StartingArtifactDownload": "Starting download {0}",
+  "UserShutdownAgent": "Agent.Listener has received a Ctrl-C or Ctrl-Break signal to shutdown.",
```

Comment:
The agent has received a shutdown signal. This can happen when the agent service is stopped, or a manually started agent is canceled.